dynawo/sources/Modeler/Common/DYNSubModel.h

```diff
@@ -1453,6 +1453,13 @@ class SubModel {
    */
   virtual void printInitValuesParameters(std::ofstream& fstream);
 
+  /**
+  * @brief write initial values internal parameters of a model in a file
```

Comment:
I would remove the "initial values" in the sentence

Reply:
done